init/sysinit.go

```diff
@@ -132,6 +132,10 @@ func runContainersFrom(startFrom string, cfg *config.Config, containerConfigs []
 				log.Errorf("Failed to run %v: %v", containerConfig.Id, container.Err)
 			}
 
+			if cfg.Debug {
+				container.Log(os.Stdout, os.Stderr)
```

Comment:
An alternative to a global debug flag could be that we have an option of which service we want to log to the console.  For example `rancher.log.containers=[network]`